integration-tests/test/native-eth.spec.ts

```diff
@@ -186,46 +189,57 @@ describe('Native ETH Integration Tests', async () => {
     await expect(
       env.l1Bridge.depositETH(DEFAULT_TEST_GAS_L2, data, {
         value: depositAmount,
-        gasLimit: 4_000_000,
       })
-    ).to.be.revertedWith(
-      'Transaction data size exceeds maximum for rollup transaction.'
-    )
+    ).to.be.reverted
   })
 
-  it('withdraw', async () => {
+  it('withdraw', async function() {
+    await useDynamicTimeoutForWithdrawals(this, env)
+
     const withdrawAmount = BigNumber.from(3)
     const preBalances = await getBalances(env)
     expect(
       preBalances.l2UserBalance.gt(0),
       'Cannot run withdrawal test before any deposits...'
     )
 
+    const transaction = await env.l2Bridge.withdraw(
+      predeploys.OVM_ETH,
+      withdrawAmount,
+      DEFAULT_TEST_GAS_L2,
+      '0xFFFF'
+    )
+    await transaction.wait()
+    await env.relayXDomainMessages(transaction)
     const receipts = await env.waitForXDomainTransaction(
-      env.l2Bridge.withdraw(
-        predeploys.OVM_ETH,
-        withdrawAmount,
-        DEFAULT_TEST_GAS_L2,
-        '0xFFFF'
-      ),
+      transaction,
       Direction.L2ToL1
     )
     const fee = receipts.tx.gasLimit.mul(receipts.tx.gasPrice)
 
     const postBalances = await getBalances(env)
 
-    expect(postBalances.l1BridgeBalance).to.deep.eq(
-      preBalances.l1BridgeBalance.sub(withdrawAmount)
+    // Approximate because there's a fee related to relaying the L2 => L1 message and it throws off the math.
+    expectApprox(
+      postBalances.l1BridgeBalance,
+      preBalances.l1BridgeBalance.sub(withdrawAmount),
+      1
```

Comment:
Slight nit out of scope of this PR - I think the third arg here should be an object that is similar to kwargs in python so its possible to know what the argument is. Its quite mysterious to see random numbers as arguments, especially when you have only looked at the implementation a few times awhile ago. Both the 3rd and 4th arg could be in that object and the keys would be descriptive of what they do. Tests should be optimized for readability so that their correctness is as easy as possible to verify

Reply:
I would recommend turning that into an issue. Would make a good first issue for someone.

Reply:
This has been turned into an issue